src/main/java/org/lantern/http/JettyLauncher.java

```diff
@@ -165,9 +168,11 @@ protected void doGet(HttpServletRequest req,
                 } else if (uri.startsWith("/proxy_google")) {
                     writeFileToResponse(resp, Proxifier.PROXY_GOOGLE);
                 } else {
-                    resp.addCookie(new Cookie("XSRF-TOKEN", model.getXsrfToken()));
-                    if (!resp.containsHeader("Content-Security-Policy")) {
-                        LanternUtils.addCSPHeader(resp);
+                    if (!LanternUtils.isDevMode() && !LanternUtils.isLanternPi()) {
```

Comment:
`isDevMode()` and `isLanternPi()` appear together a lot.  Can/should they be distilled into a single flag?